Update FAQ-UseWasabi.md to include MinGapLimit (#217)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

![](/ReceiveLabelingRequired.png)
:::

:::details
### How do I generate multiple new receiving addresses?

> Wasabi as default allows up to 21 unused addresses, you see this reflected in the graphical interface, that shows by default max 21 addresses. After that, it'll start to repropose you the same addresses, this is to avoid bloating of the software. More info [here](https://github.com/zkSNACKs/WalletWasabi/issues/2340#issuecomment-534885887).  

Reviewer: ```suggestion
Wasabi by default allows up to 21 unused addresses, you see this reflected in the graphical interface, that shows max 21 addresses.
After that, it'll regenerate the same addresses, this is to avoid bloating the software.
More info [here](https://github.com/zkSNACKs/WalletWasabi/issues/2340#issuecomment-534885887).  
```

Let's see an example:

`m/84'/0'/0'` is the dafault BIP84 derivation path, as explained [here](FAQ-UseWasabi.md#What-derivation-paths-does-Wasabi-use), and so this'll be your first receiving address: `m/84'/0'/0'/0/0`.  
`m/84'/0'/0'/0/1` this will be the second.  
`m/84'/0'/0'/0/2` this will be the third, and so on...

> At `m/84'/0'/0'/0/20` (your 21st address), if you never used any of the previous generated addresses, Wasabi will start recycling the previous unused ones and will automatically overwrite the labels. e.g., Wasabi will not generate address `m/84'/0'/0'/0/21` with label: "address n. 22", instead it will repropose one between `m/84'/0'/0'/0/0` and `m/84'/0'/0'/0/20` and will overwrite previous label to "address n. 22".

Reviewer: ```suggestion
At `m/84'/0'/0'/0/20` (your 21st address), if you never used any of the previously generated addresses, Wasabi will start recycling the previous unused ones and will automatically overwrite the labels. e.g., Wasabi will not generate address `m/84'/0'/0'/0/21` with label: "address n. 22", instead it'll regenerate one between `m/84'/0'/0'/0/0` and `m/84'/0'/0'/0/20` and will overwrite previous label to "address n. 22".
```
